src/busctl/busctl.c

```diff
@@ -167,7 +167,9 @@ static int list_bus_names(int argc, char **argv, void *userdata) {
         if (r < 0)
                 return log_error_errno(r, "Failed to list names: %m");
 
-        pager_open(arg_no_pager, false);
+        r = pager_open(arg_no_pager, false);
```

Comment:
Hmm, do we really want this? It adds verbosity. On one hand, it's OK if the pager does not run, on the other hand, it seems correct to check if forking failed. I guess it *is* more correct to have the check.